jbpm-audit/src/test/java/org/jbpm/process/audit/jms/AsyncAuditLogProducerTest.java

```diff
@@ -233,12 +235,19 @@ public void testAsyncAuditLoggerComplete() throws Exception {
         List<ProcessInstanceLog> processInstances = logService.findProcessInstances("com.sample.ruleflow");
         Assertions.assertThat(processInstances.size()).isEqualTo(1);
         List<NodeInstanceLog> nodeInstances = logService.findNodeInstances(processInstance.getId());
+        Collections.sort(nodeInstances, (o1, o2) -> o1.getDate().compareTo(o2.getDate()));
```

Comment:
May getId be better to use in comparison than getDate ?  Anyway, is it needed to sort the collection?

Reply:
that line was  not needed (removed)